src/index.js

```diff
@@ -66,103 +45,64 @@ export default {
         return this._getDeviceFromDetails(browserDetails) !== null;
     },
 
-    // Extra methods
     async resizeWindow (/* id, width, height, currentWidth, currentHeight */) {
-        this.reportWarning('The window resize functionality is not supported by the "fbsimctl" browser provider.');
+        this.reportWarning('The window resize functionality is not supported by the "ios" browser provider.');
     },
 
     async takeScreenshot (id, screenshotPath) {
-        var command = 'xcrun simctl io ' + this.currentBrowsers[id].udid + ' screenshot \'' + screenshotPath + '\'';
+        var command = `xcrun simctl io ${this.currentBrowsers[id].udid} screenshot \'${screenshotPath}\'`;

```

Comment:
👍  to templates

You don't need the `\'` you can just hae inline single-quotes